File: 1546 평균/1546.js
let fs = require('fs');
let inputs = fs.readFileSync('/dev/stdin').toString().split('\n');

let n = parseInt(inputs[0]);
let scoreArray = inputs[1].split(' ');
let maxScore = Math.max(...scoreArray);
scoreArray = scoreArray.map((value)=>value/maxScore * 100);

console.log(scoreArray.reduce((prev,current)=> prev + current) / n);

File: 1546 평균/1546.java
import java.io.*;
import java.util.Scanner;

public class Main {
    static public void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        double[] scoreArray = new double[n];
        double max = -987654321;
        for(int i=0;i<n;++i){
            scoreArray[i] = scanner.nextDouble();
            max = Math.max(scoreArray[i], max);
        }

        for(int i=0;i<n;++i){
            scoreArray[i] = scoreArray[i] / max * 100;
        }

        double sum = 0;
        for(int i=0;i<n;++i){
            sum += scoreArray[i];
        }

        System.out.println(sum/n);
    }
}
